dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -537,29 +537,34 @@ RtpsUdpDataLink::leave_multicast_group(const NetworkInterface& nic)
 
 void
 RtpsUdpDataLink::add_locators(const RepoId& remote_id,
-                              const ACE_INET_Addr& narrow_address,
-                              const ACE_INET_Addr& wide_address,
+                              const AddrSet& narrow_addresses,
+                              const AddrSet& wide_addresses,
```

Comment:
narrow/wide may be confusing - should these be renamed?

Reply:
We discussed this with @simpsont-oci when this code first went in.  What was the motivation for not using unicast and multicast?

Reply:
The general motivation was because "multicast_address" makes no sense when multicast is disabled. I agree, the names are still potentially confusing. Perhaps undirected / directed or spread / focused or something similar?

Reply:
My thought is that a multicast address is still a multicast address, even if we are not using it.

I've updated the code is a couple places to accommodate this.  First, when we are extracting locators from the blob, multicast addresses will be skipped in multicast is disabled.  Second, when we get to the point of selecting addresses, we1
1. Use the unicast address if preferred and available.
2. Use the multicast address if available.
3. Fall back to the unicast address.

So, 1 and 3 together means they when multicast is disable, they will always get the unicast locators.

Reply:
> My thought is that a multicast address is still a multicast address, even if we are not using it.

The previous code was setting a variable named "multicast_address" to be a unicast address.